src/pages/notification/index.tsx

```diff
@@ -175,6 +175,7 @@ const NoEmptyNotificationText = styled.span`
 
 interface NotificationItemScheme {
   landingUrl: string;
+  deeplinkUrl?: string | null;
```

Comment:
`deeplinkUrl` 필드가 아예 없을 수도 있는거죠?

Reply:
```suggestion
  deeplinkUrl: string | null;
```
스토어팀 스펙이 optional 에서 nullable 로 변경됭 것 같습니다.
둘다 있을 수 있다고 가정하고 넣었는데, 타입스크립트에서는 nullable 로 고정해도 될 것 같아요.